src/loader.py

```diff
@@ -219,88 +212,12 @@ def merge_summary_documents(summaries, metadata_list):
 ################################################################################################
 
 
-def get_file_summary(path: str):
-    client = Groq(
-        api_key=os.environ.get("GROQ_API_KEY"),
-    )
+def get_file_summary(path: str, incognito=True):
     reader = SimpleDirectoryReader(input_files=[path]).iter_data()
 
     docs = next(reader)
     splitter = TokenTextSplitter(chunk_size=6144)
     text = splitter.split_text("\n".join([d.text for d in docs]))[0]
     doc = Document(text=text, metadata=docs[0].metadata)
-    summary = dispatch_summarize_document_sync(doc, client)
```

Comment:
all these sync methods should come back for watch_util; or be replaced by a sync wrapper around the async ones (I got rid of these out of naivety and a dislike of duplication)